preview_generator/preview/builder/document_generic.py

```diff
@@ -143,20 +143,17 @@ def build_pdf_preview(
         if page_id < 0:
             return  # in this case, the intermediate file is the requested one
 
-        pdf_in = utils.get_decrypted_pdf(intermediate_pdf_file_path)
-        output_file_path = os.path.join(cache_path, "{}{}".format(preview_name, extension))
         pdf_out = PdfFileWriter()
-        pdf_out.addPage(pdf_in.getPage(page_id))
+        with open(intermediate_pdf_file_path, "rb") as pdf_stream:
+            pdf_in = utils.get_decrypted_pdf(pdf_stream)
```

Comment:
regression found here: 
for unknown reason, using stream instead of file path in get_decrypted_pdf() which use PdfFileReader inside doesn't provide a correct pdf (it result in an empty pdf).
see https://github.com/algoo/preview-generator/issues/194